### DISCUSSION TITLE
The urls inside the composer.json should be in their secure version?

### DISCUSSION THREAD

**Discussion by S-Tian86** [doctrine/DoctrineBundle#1440] [Q&A] [answered]
Hi, 

I think these urls should be written with https:// instead of http://.
https://github.com/doctrine/DoctrineBundle/blob/f07b2ed3ee6bfac4a97d36eb9dfa004157981f05/composer.json#L6
https://github.com/doctrine/DoctrineBundle/blob/f07b2ed3ee6bfac4a97d36eb9dfa004157981f05/composer.json#L18-L21
https://github.com/doctrine/DoctrineBundle/blob/f07b2ed3ee6bfac4a97d36eb9dfa004157981f05/composer.json#L22-L25

What do you think?

Regards

**Comment by greg0ire** [comment]
Yes.

**Reply by S-Tian86** [reply]
Here are some other urls in other repository that are also without https.

https://github.com/doctrine/DoctrineMigrationsBundle/blob/cb20337b74d652ad2a2e180557224ff986acacd6/composer.json#L13-L20
```json
  {
      "name": "Doctrine Project",
      "homepage": "http://www.doctrine-project.org"
  },
  {
      "name": "Symfony Community",
      "homepage": "http://symfony.com/contributors"
  }
```
https://github.com/doctrine/sql-formatter/blob/89a82b90e7c44149fb765cde5f4f9fe75cd73107/composer.json#L15-L19
```json
  {
      "name": "Jeremy Dorn",
      "email": "jeremy@jeremydorn.com",
      "homepage": "http://jeremydorn.com/"
  }
```
https://github.com/doctrine/data-fixtures/blob/43ae7806b77ce75b792337c90250ba502eedb84e/composer.json#L8
```json
  "homepage": "http://www.doctrine-project.org",
```
https://github.com/doctrine/DoctrineFixturesBundle/blob/cbc40e6cde6bc435b61e2475d3e0d04c2308c8b8/composer.json#L6-L21
```json
  "homepage": "http://www.doctrine-project.org",
  "license": "MIT",
  "authors": [
      {
          "name": "Fabien Potencier",
          "email": "fabien@symfony.com"
      },
      {
          "name": "Doctrine Project",
          "homepage": "http://www.doctrine-project.org"
      },
      {
          "name": "Symfony Community",
          "homepage": "http://symfony.com/contributors"
      }
  ],
```
Regards.

**Comment by greg0ire** [comment, marked as answer]
Why don't you send PRs? :)

**Reply by S-Tian86** [reply]
Sure, I will send you a PR for each one.

**Reply by S-Tian86** [reply]
It's okay, PRs sent
